apps/emqx_dashboard/src/emqx_dashboard_listener_config.erl

```diff
@@ -134,26 +131,6 @@ remove_sensitive_data(Conf0) ->
             Conf1
     end.
 
-post_config_update(_, {change_i18n_lang, _}, _NewConf, _OldConf, _AppEnvs) ->
-    delay_job(regenerate);
-post_config_update(_, _Req, NewConf, OldConf, _AppEnvs) ->
-    SwaggerSupport = diff_swagger_support(NewConf, OldConf),
-    OldHttp = get_listener(http, OldConf),
-    OldHttps = get_listener(https, OldConf),
-    NewHttp = get_listener(http, NewConf),
-    NewHttps = get_listener(https, NewConf),
-    {StopHttp, StartHttp} = diff_listeners(http, OldHttp, NewHttp, SwaggerSupport),
-    {StopHttps, StartHttps} = diff_listeners(https, OldHttps, NewHttps, SwaggerSupport),
-    Stop = maps:merge(StopHttp, StopHttps),
-    Start = maps:merge(StartHttp, StartHttps),
-    delay_job({update_listeners, Stop, Start}).
-
-%% in post_config_update, the config is not yet persisted to persistent_term
-%% so we need to delegate the listener update to the gen_server a bit later
-delay_job(Msg) ->
```

Comment:
This looks quite hairy (what if config update lasts longer?), we try to make this a bit more accurate (see `emqx_dashboard_dispatch:regenerate_dispatch_after_config_update/1`)